src/index.ts

```diff
@@ -60,12 +105,86 @@ async function encryptWithKey<R>(
  * the resulting value
  * @param {string} password - password to decrypt with
  * @param {string} text - cypher text to decrypt
+ * @param {CryptoKey} key - a key to use for decrypting
+ * @returns {object}
  */
-async function decrypt<R>(password: string, text: string): Promise<R> {
+async function decrypt(
+  password: string,
+  text: string,
+  key: CryptoKey,
```

Comment:
This seems to be optional as well

```suggestion
  key?: CryptoKey,
```